templates_jinja2/suggest_offseason_event.html

```diff
@@ -0,0 +1,88 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Add Offseason Event{% endblock %}
+
+{% block meta_description %}Add an offseason event to The Blue Alliance.{% endblock %}
+
+{% block content %}
+    <div class="container">
+        {% if status %}
+        <div class="row">
+            <div class="col-xs-12 col-lg-8 col-lg-offset-2">
+                {% if status == 'success' %}
+                <div class="alert alert-success">
+                    <button type="button" class="close" data-dismiss="alert">&times;</button>
+                    <h4>Thanks!</h4>
+
+                    <p>We'll review your suggestion and get it added to the site soon!</p>
+                </div>
+                {% elif status == 'missing_data' %}
+                <div class="alert alert-danger">
+                    <button type="button" class="close" data-dismiss="alert">&times;</button>
+                    <h4>Uh Oh!</h4>
+
+                    <p>Please ensure you've entered valid data for all fields.</p>
+                </div>
+                {% endif %}
+            </div>
+        </div>
+        {% endif %}
+        <div class="row">
+            <div class="col-xs-12 col-lg-8 col-lg-offset-2">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h1 class="panel-title">Let us know about offseason events!</h1>
+                    </div>
+                    <div class="panel-body">
+                        <p>Thanks for helping make The Blue Alliance better! Let us know about
+                            offseason events so we can add them to the site!</p>
+                        <ul>
+                            <li>Your suggestion will be reviewed by a moderator</li>
+                            <li>Your account info (like {{ user_bundle.user.email() }}
+                                and {{ user_bundle.user.nickname() }}) will also be submitted. <a
+                                        href="{{ user_bundle.logout_url }}">log in as a different
+                                    user</a></li>
+                            <li>All fields below are required, please do the best you can to fill them accurately</li>
+                            <li>You can only suggest offseason events, please <a href="/contact">contact us</a> you find errors with official regional and district events on the site</li>
+                        </ul>
+                    </div>
```

Comment:
I think you closed the panel body too early. The "Suggest an Event" header and form should be enclosed.